tests/test_lane.py

```diff
@@ -1446,6 +1446,15 @@ def test_initialize(self):
         # to the constructor.
         eq_([1,2,3], wl.entrypoints)
 
+    def test_initialize_without_library(self):
+        # It's possible to initialize a WorkList with no Library.
+        worklist = WorkList()
+        worklist.initialize(None)
+
+        # No restriction is placed on the collection IDs of the
+        # Works in this list.
+        eq_(None, worklist.collection_ids)
+
```

Comment:
This fixes a problem I saw in circulation: a CrawlableCollectionLane was initialized with a collection ID instead of a library. That worked fine, but then a SpecificWorkList was initialized based on the (nonexistent) library of the CrawlableCollectionLane. The behavior should have been "get these works regardless of which collections they're in", but it was "get these works but only if they're in the null set of collections", which was matching nothing.